setup/providers/airvpn.md

```diff
@@ -55,6 +59,14 @@ services:
       - ./gluetun:/gluetun
 ```
 
+## VPN port forwarding
+
+If you want to use VPN server side port forwarding:
+
+1. Log in your AirVPN account at [airvpn.org/client](https://airvpn.org/client/)
+1. Obtain a port from [airvpn.org/ports](https://airvpn.org/ports/)
+1. Add the port to the environment variable `FIREWALL_VPN_INPUT_PORTS`
+
```

Comment:
Please revert to have this at the bottom, this is 'less important' than the sections below it.